helm-charts/yunikorn/templates/admission-controller-deployment.yaml

```diff
@@ -0,0 +1,91 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: yunikorn-admission-controller
+  labels:
+    app: yunikorn
+    chart: {{ include "yunikorn.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.admissionControllerReplicaCount }}
+  selector:
+    matchLabels:
+      app: yunikorn
+      component: yunikorn-admission-controller
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      name: yunikorn-admission-controller
+      labels:
+        app: yunikorn
+        component: yunikorn-admission-controller
+        release: {{ .Release.Name }}
+    spec:
+      serviceAccountName: {{ .Values.admissionControllerServiceAccount }}
+      containers:
+        - name: yunikorn-admission-controller
+          image: "{{ .Values.admission_controller_image.repository }}:{{ .Values.admission_controller_image.tag }}"
```

Comment:
The variable name should be in camelcase instead of being separated by underscores according to the [Helm convention](https://helm.sh/docs/chart_best_practices/values/#naming-conventions)